sdk/storage/storage-blob/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2019.11 12.0.0-preview.6
+
+- Bug Fix - Previous versions of `@azure/storage-blob` preview library failed for the react-apps because of the usage of `fs.stat` method which is not available in browsers and due to the presence of some circular dependencies.  Both the issues are fixed in this new release.
```

Comment:
```suggestion
- Bug Fix - Previous versions of `@azure/storage-blob` preview library failed for React apps because of the usage of `fs.stat` method which is not available in browsers and due to the presence of some circular dependencies.  Both of these issues are fixed in this new release.
```